content/beitragen/als-organisation.md

```diff
@@ -45,9 +45,9 @@ will.
 [OSM-Zugang einrichten](/beitragen/osm-zugang/)
 {.link-intern}
 
-## Organised Editing Guidelines
```

Comment:
Wir benutzen in der Regel bei OSM das britische Englisch (siehe auch den Link zur offiziellen Version), daher passt das schon so mit dem "organised".